modules/mobile/docs/openapi.yaml

```diff
@@ -1308,6 +1308,41 @@ paths:
       security:
         - Bearer: []
       summary: /v0/community-care-providers
+  /v0/dependents:
+    get:
+      description: Returns a list of user's dependents. Dependent SSN is also available but is removed to limit PII exposure.
```

Comment:
@aherzberg Per discussion, I removed the SSN but mentioned that it's available in the docs.